### PR TITLE
CO-1552

### PR DESCRIPTION
**Pull request by vwvw**

### CODE REVIEW

File: hr_attendance_calendar/views/hr_attendance_day_view.xml [hunk 1]
```diff
@@ -42,15 +42,19 @@
                                 <tree>
                                     <field name="check_in"/>
                                     <field name="check_out"/>
-                                    <field name="worked_hours" widget="float_time"/>
+                                    <field name="worked_hours"
```
Comment (ecino) [inline hr_attendance_calendar/views/hr_attendance_day_view.xml]: paid_hours?

File: hr_attendance_calendar/models/hr_attendance_day.py [hunk 2]
```diff
@@ -146,9 +148,9 @@ def _compute_due_hours(self):
 
     @api.multi
     @api.depends('attendance_ids.worked_hours')
-    def _compute_worked_hours(self):
+    def _compute_paid_hours(self):
         for att_day in self.filtered('attendance_ids'):
-            worked_hours = sum(att_day.attendance_ids.mapped('worked_hours'))
+            logged_hours = sum(att_day.attendance_ids.mapped('paid_hours'))
```
Comment (ecino) [inline hr_attendance_calendar/models/hr_attendance_day.py]: I think in model `hr.attendance` the field worked_hours is still existing so here it should be `logged_hours = sum(att_day.attendance_ids.mapped('worked_hours'))`

File: hr_attendance_calendar/views/hr_attendance_day_view.xml (hunk 1 above)
Comment (vwvw) [reply hr_attendance_calendar/views/hr_attendance_day_view.xml]: Nope. This refers to the worked_hours field from the attendance class.

**Comment by ecino** [comment]
@eicher31 please review
@vwvw apparently there are conflicts with your previous PR

**Review by eicher31 — APPROVED** [review]

**Pull request merged**